x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/80_text.yml

```diff
@@ -385,8 +385,28 @@ setup:
   - length: { values: 2 }
   - match: { values.0: [ [ "foo", "bar" ] ] }
   - match: { values.1: [ "baz" ] }
+---
+"reverse text":
+  - requires:
+      cluster_features: ["gte_v8.14.0"]
```

Comment:
I don't think this is right. I _think_ this needs to mirror what you added in EsqlCapabilities. Something like `cluster_features: ["esql.fn_reverse"]`. @nik9000 or @not-napoleon to confirm.

Otherwise this would break mixed versions/mixed clusters tests.

Reply:
Good point. The version is certainly wrong. But adding `esql.fn_reverse` results in

```
java.lang.IllegalArgumentException: Unknown feature esql.fn_reverse: check the respective FeatureSpecification is provided both in module-info.java as well as in META-INF/services and verify the module is loaded during tests.
```

Reply:
maybe I just go with 8.16?

Reply:
Ah! Sorry, this is an old pattern. Try:
```
  - requires:
      capabilities:
        - method: POST
          path: /_query
          parameters: [method, path, parameters, capabilities]
          capabilities: [fn_reverse]
      reason: "reverse not yet added"
```


Sorry about this one. It's the "new way" to mark new things. It turns out using cluster features for this was a mistake - though one we only learned was a mistake by making the mistake for a month or two.

Reply:
I made this change in https://github.com/elastic/elasticsearch/pull/113297/commits/5bfcd0ad97206e4b12840908ec25533077e24f2f, but I'm worried that the test is no longer running.

When I run `./gradlew :x-pack:plugin:esql:qa:server:single-node:yamlRestTest --tests "org.elasticsearch.xpack.esql.qa.single_node.EsqlClientYamlIT.test {p0=esql/80_text/reverse text}"` it says it can't find a matching test....

Reply:
I'm pulling it locally to have a look.